frontend/src/app/testQueue/TestCardForm/TestCardForm.tsx

```diff
@@ -279,24 +270,53 @@ const TestCardForm = ({
     return null;
   };
 
-  const getAOEError = () => {
+  const whichAOEFormOption = useAOEFormOption(state.deviceId, devicesMap);
+  const whichAoeFormRequiredForPositives =
+    state.testResults.some((x) => x.testResult === TEST_RESULTS.POSITIVE) &&
+    whichAOEFormOption !== AOEFormOption.COVID
+      ? whichAOEFormOption
+      : null;
+  const areAoeQuestionsComplete = () => {
```

Comment:
nit: Could we rename this one to something like `validateAoeForm` since this function is not only returning a boolean, but also handling whether to show any errors?

And then for the `validateAoeQuestions` function, could that be renamed to `areAoeQuestionsAnswered` especially since within that function we use helpers also named `are[Non]SymptomAoeQuestionsAnswered`